rinad/src/ipcm/main.cc

```diff
@@ -203,6 +208,11 @@ int main(int argc, char * argv[])
         }
         LOG_DBG("SIGPIPE handler installed successfully");
 
+	if (signal(SIGCHLD, handler) == SIG_ERR) {
+		LOG_ERR("Could not install SIGCHLD handler!");
+	}
```

Comment:
I followed the same style as the other ifs that were already in this method. Should all of them be updated then?